demo/Gemfile

```diff
@@ -25,6 +25,9 @@ gem "high_voltage", "~> 3.1"
 # Bundle the design system engine
 gem "citizens_advice_components", path: "../engine"
 
+## Explicitly pin to ViewComponent 2 for the demo app
+gem "view_component", "~> 2.82.0"
```

Comment:
This is required as whilst the engine now supports view component 3 not all of the internal usages are updated. You'd probably need to do this for existing apps that have their own components.

Reply:
Ah I just meant the code in https://github.com/citizensadvice/design-system/tree/main/demo